newIDE/app/scripts/extract-extensions-document.js

```diff
@@ -164,27 +236,103 @@ const createExtensionReferencePage = async (extensionHeader, isCommunity) => {
   console.info(`ℹ️ File generated: ${extensionReferenceFilePath}`);
 };
 
+/**
+ * Create a page for an extension.
+ * @param {any} gd
```

Comment:
Arthuro555 PR to have TypeScript typings for gd will be super helpful for these scripts 👍